### PR TITLE
Update react native reanimated to 2.5.0

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Update react native reanimated to 2.5.0 to fix crashes on lower tier Android devices.

### Other changes

N/A

### Tested

WIP

### How others should test

Launch the app in release mode on Android 5 - 6 OS.

### Related issues

N/A

### Backwards compatibility

Yes

### CODE REVIEW

**Review by gnardini — APPROVED** [review]
This isn't the first time this happens, I wonder if there's a way to completely disable animations in older versions 🤔

**Comment by MuckT** [comment]
### Additional Details
- [React Native Reanimated PR](https://github.com/software-mansion/react-native-reanimated/pull/2847)
- [Our Sentry Issue](https://sentry.io/organizations/valora-inc/issues/2987042561)

File: packages/mobile/src/navigator/QRNavigator.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ function AnimatedScannerScene({ route, position, ...props }: AnimatedScannerScen
   )
 
   return (
-    <Animated.View style={animatedStyle}>
+    <Animated.View style={[animatedStyle]}>
```
Comment (jeanregisser) [inline packages/mobile/src/navigator/QRNavigator.tsx:115]: Looks like this is just silencing a TS error?

**Review by jeanregisser — APPROVED** [review]
Thanks! 🙏

File: packages/mobile/src/navigator/QRNavigator.tsx (hunk 1 above)
Comment (MuckT) [reply packages/mobile/src/navigator/QRNavigator.tsx:115]: Without the array it wasn't building. I do see use memo used for animated styles in other places so I am not sure why this started failing with the update.

Comment (jeanregisser) [reply packages/mobile/src/navigator/QRNavigator.tsx:115]: ok then can you change it to? This way it's a bit more clear there's a TS issue.
```suggestion
    // @ts-expect-error
    <Animated.View style={animatedStyle}>
```

Comment (MuckT) [reply packages/mobile/src/navigator/QRNavigator.tsx]: Fixed in https://github.com/valora-inc/wallet/pull/2230/commits/3786a5a828713cd1827e313eb9cde92c14ee9f40!

**Comment by MuckT** [comment]
@Mergifyio refresh

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Android Internal Release Build V1.29.0 & updated status in test rails.

-  User is able to install the app and app is not crashing.

**Test rail link:**
**Android 11:** https://valoraapp.testrail.io/index.php?/tests/view/20949

**Device:** Lg K7i(6.0)

Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Android Internal Release Build V1.29.2 & updated status in test rails.

User is able to install the app and app is not crashing.
Test rail link:
Android 5: https://valoraapp.testrail.io/index.php?/runs/view/94&group_by=cases:section_id&group_order=asc
Device: Samsung Galaxy A3(5.0.2)

Thanks..!!

**Pull request merged**